src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/recommendation/RecommendationServiceTest.kt

```diff
@@ -177,6 +178,7 @@ internal class RecommendationServiceTest : ServiceTestBase() {
         agencyId = "BRX",
         facialImageId = null,
         locationDescription = "Outside - released from Leeds",
+        releaseDate = LocalDate.parse("2025-01-01"),
```

Comment:
Depending on the answer to the earlier point, this may need to be replaced with a `SentenceDetail` value